kodilla-stream/src/test/java/com/kodilla/exception/homework/WarehouseTestSuite.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.exception.homework;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class WarehouseTestSuite {
+
+    @Test
+    public void testGetOrder_withException() {
+        Throwable exception = assertThrows(
+                OrderDoesntExistException.class, () -> {
+                    Warehouse.addOrder(new Order("1"));
```

Comment:
To jest test jednostkowy, tak? To tutaj testujesz metody addOrder i getOrder. Jak wywali sie exception, to skad wiesz, ktora metoda i z jakim argumentem rzucila wyjatek?